SU-SWS/acquia_purge_d8cache
# Disable only cache settings form in views

# READY FOR REVIEW

# Summary
- Disable views advanced settings only for the cache area.
- Don't disabled advanced form for other settings.

# Needed By (Date)
- ASAP

# Urgency
- HIGH

# Steps to Test
1. checkout this branch
1. edit a view
1. click to edit any of the settings under "Advanced"
1. verify you are allowed to edit
1. click to edit the "Cache" settings
1. verify you are not allowed to edit those settings.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
